### PR TITLE
Methodology component

### PR DESCRIPTION
**Pull request by nbarrerabs**
# FooCamp - Bardo

## Trello ticket
https://trello.com/c/ZR6SkNDZ/54-que-hacemos-metodolog%C3%ADa-funcionalidad-dise%C3%B1os-sp2

## Description
Create the Methodology component reusing Hero components

## To reproduce

1. Run the project
2. Open the specific page http://localhost:300/que-hacemos
3. Check the component

## Screenshots
Desktop
image

Mobile
image

### CODE REVIEW

**Review by dyepes62 — APPROVED** [review]

File: styles/components/_methodology.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  &__container {
+    display: grid;
+    grid-column: auto;
+
+    &--child {
+      display: grid;
+      grid-template-columns: repeat(12, 1fr);
+      grid-column-gap: 24px;
+      margin-right: 5.3%;
```
Comment (SinuheM) [inline styles/components/_methodology.scss]: I think this margin does not help to center the element

File: styles/components/_methodology.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      text-align: center;
+    }
+  }
+
+  &__body-wrapper {
+    &--child {
+      padding: 16px 0 0 0;
+      max-width: 1058px;
+      max-height: 129px;
+      grid-column: 3/12;
```
Comment (SinuheM) [inline styles/components/_methodology.scss]: Follow the previous comment if it needs a little space the margin could be here to both sides, and the `grid-column: 2/12`

File: styles/components/_methodology.scss (hunk 1 above)
Comment (nbarrerabs) [reply styles/components/_methodology.scss]: This is not to center is to align with the image on Impact component:
image

Also, I copied from your style impact-level__image :)

File: styles/components/_methodology.scss (hunk 2 above)
Comment (nbarrerabs) [reply styles/components/_methodology.scss]: Same as above, this is for align to Impact component, you also have on impact-level__image class **grid-colum: 3/4**

File: styles/components/_methodology.scss (hunk 1 above)
Comment (SinuheM) [reply styles/components/_methodology.scss]: Oh! ups, my bad; its so cool it's aligned to the image :) ... but I think it's margin must to be on both sides

Comment (SinuheM) [reply styles/components/_methodology.scss]: image

File: styles/components/_methodology.scss (hunk 2 above)
Comment (SinuheM) [reply styles/components/_methodology.scss]: Yes sir, it have a extra space, it's aligned to the right

**Comment by nbarrerabs** [comment]
@SinuheM How you see it now ?

image

File: styles/components/_methodology.scss [hunk 3]
```diff
@@ -15,9 +15,10 @@
       display: grid;
       grid-template-columns: repeat(12, 1fr);
       grid-column-gap: 24px;
-      margin-right: 5.3%;
+      margin-right: 5.8%;
       justify-content: center;
       text-align: center;
+      margin-left: 78px;
```
Comment (SinuheM) [inline styles/components/_methodology.scss]: Maybe this margin at 5.8% like the margin-right, and It's going to be great :)

**Review by SinuheM — APPROVED** [review]

**Pull request merged**